src/utils.js

```diff
@@ -535,6 +535,30 @@ function serializeKeysForMessage(keys, maxLength) {
   return '';
 }
 
+function sanitize(input, sanitizeKeys) {
+  sanitizeKeys = isArray(sanitizeKeys) ? sanitizeKeys : [];
```

Comment:
if it's not an array, maybe just bail out here and return array? to reduce the risk involved in shipping this code?